io/subsystem/src/main/resources/org/wildfly/extension/io/LocalDescriptions.properties

```diff
@@ -14,9 +14,9 @@ io.worker.task-keepalive=Specify the number of milliseconds to keep non-core tas
 io.buffer-pool=Defines buffer pool
 io.buffer-pool.add=Adds new buffer pool
 io.buffer-pool.remove=Removes buffer pool
-io.buffer-pool.buffers-per-slice=How many buffers per slice, if not set optimal value is calculated based on available RAM resources in your system.
-io.buffer-pool.buffer-size=The size of each buffer slice in bytes, if not set optimal value is calculated based on available RAM resources in your system.
-io.buffer-pool.direct-buffers=Does the buffer pool use direct buffers, some platforms don't support direct buffers
+io.buffer-pool.buffers-per-slice=Maximum number of buffers
+io.buffer-pool.buffer-size=Total size of the buffer handled by the subsystem
```

Comment:
This isn't the total size of the buffer. The existing comment is valid.

The basic problem here is "buffers-per-slice" would be better named "slices-per-buffer" and "buffer-size" would be better named "slice-size". Or, ignore the 'slice' concept and instead describe and refer to the 'region' concept, using 'buffers-per-region' and 'buffer-size'. The "buffers-per-slice" is particularly problematic as it is actually backwards in terms of what the attribute does.

Basically there is a larger backing byte buffer which is the unit of memory allocation from the JVM. Then there are smaller chunks of that that are provided to users of this resource. The larger backing buffer can be called 'buffer' and the smaller chunks are 'slices' or the larger backing buffer is a 'region' and the smaller chunks are 'buffers'.

The size of the larger backing buffer is buffers-per-slice * buffer-size.

We can't drop the existing attributes and if new better ones are added it's necessary to add a transformer to ensure that older version hosts in a managed domain are not broken. Probably better for the next release as we are about to only accept blockers.